### PR TITLE
Kademlia network and Tendermint v0.32.0 

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -5,7 +5,7 @@ docstrings = JavaDoc
 maxColumn = 120
 
 align = some
-align.tokens = [{code = "=>", owner = "Case"}]
+align.tokens = [{code = "=>", owner = "Case"}, ":=", "%", "%%"]
```
Comment (alari) [inline .scalafmt.conf]: Add `%%%` for scala.js.

File: kademlia/http/src/main/scala/fluence/kad/http/KademliaHttp.scala [hunk 2]
```diff
@@ -81,7 +81,7 @@ class KademliaHttp[F[_]: Sync, C](
 
           }
 
-        case req @ POST -> Root / "ping" ⇒
+        case req @ (POST | GET) -> Root / "ping" ⇒
```
Comment (alari) [inline kademlia/http/src/main/scala/fluence/kad/http/KademliaHttp.scala]: Actually `ping` is a mutating action: it promotes a pinging node to a pingee. So we could leave `GET` for now, but later on, we should add a `status` endpoint just to read current Kad status.

File: log/src/main/scala/fluence/log/Context.scala [hunk 3]
```diff
@@ -37,8 +37,8 @@ case class Context(data: Map[String, String] = Map.empty, loggingLevel: Log.Leve
 
 object Context {
 
-  def init(k: String, v: String = "", level: Log.Level = Log.Info): Context =
-    Context(Map(k -> v), level)
+  def init(kv: (String, String)*)(level: Log.Level = Log.Info): Context =
```
Comment (alari) [inline log/src/main/scala/fluence/log/Context.scala]: Let's make two different `init` methods instead.

File: node/src/main/resources/docker/application.conf [hunk 4]
```diff
@@ -41,4 +41,16 @@ worker {
...
 
-kademlia.advertize.host = ${TENDERMINT_IP}
+kademlia {
+  advertize {
+    host = ${EXTERNAL_IP}
+    port = ${API_PORT}
+  }
+
+  join {
+    seeds = [
```
Comment (alari) [inline node/src/main/resources/docker/application.conf]: Meh, should it be there? Could we move the seeds to deployment configuration?

File: log/src/main/scala/fluence/log/Context.scala (hunk 3 above)
Comment (folex) [reply log/src/main/scala/fluence/log/Context.scala]: unfortunately there can't be two overloaded methods with default arguments

**Pull request merged**